vector/src/main/res/values/strings.xml

```diff
@@ -1294,6 +1294,11 @@ Why choose Riot.im?
     <string name="keys_backup_setup_step3_save_button_title">Save as File</string>
     <string name="recovery_key_export_saved_as_warning">The recovery key has been saved to \'%s\'.\n\nWarning: this file may be deleted if the application is uninstalled.</string>
 
+    <string name="keys_backup_setup_override_backup_prompt_tile">A backup already exist in this location</string>
```

Comment:
A backup already exist**s** ~~in this location~~ **on your homeserver**

Reply:
done